hardware/is_camera/src/camera.cc

```diff
@@ -243,6 +245,25 @@ namespace is_camera {
     thread_ = std::thread(&CameraNodelet::PublishLoop, this);
   }
 
+  // Set the exposure
+  bool CameraNodelet::SetExposure(ff_msgs::SetExposure::Request  &req,
+                          ff_msgs::SetExposure::Response &res) {
+    if (thread_running_ && !auto_exposure_) {
+      // Set exposure
+      camera_exposure_ = req.exposure;
+      v4l_->SetParameters(camera_gain_, camera_exposure_);
+      // Success!
+      res.success = true;
+      res.status_message = "Success";
+    } else {
+      // Failed
+      res.success = false;
+      res.status_message = "Failed! Either thread not running or auto-exposure on";
```

Comment:
It would probably be better to include the values of `thread_running_` and `auto_exposure_` in this message.